views/js/qtiCreator/test/MathEditor/test.js

```diff
@@ -0,0 +1,67 @@
+define([
```

Comment:
So for now I'm not moving my test as it will be moved with the rest of the folder or shall I move it already ?